### PR TITLE
Map polish follow up

### PR DESCRIPTION
**Pull request by gemfarmer**
Fixes issue(s) #1938, other assorted issues

[:sunglasses: PREVIEW](https://federalist.18f.gov/preview/18F/doi-extractives-data/homepage-map-polish-followup/)

Changes proposed in this pull request:
- center-aligned map legend on explore and homepage
- Updated the selector to include 'Nationwide data' as an option, default option
- addressed #1938

/cc @ericronne @coreycaitlin

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -23,14 +23,18 @@ <h3 class="ribbon-hero-description-light">
...
       </p>
       <figcaption class="state_pages-select">
-        <label for="location-selector">Choose a state or offshore region</label>
-        {% include selector.html %}
+        <label for="location-selector">Explore data</label>
+        {% include selector.html
+           nationwide = true
+           nationwide_url = '/explore/'
+           nationwide_title = 'Nationwide data'
+           default = 'Choose one (COREY???)'
```
Comment (gemfarmer) [inline index.html]: @coreycaitlin we were struggling to pick a default set of text here. To be consistent with the previous layout, we would want `Choose a state or offshore region`, but the width of the selector doesn't fully accommodate that text. Any ideas for how to succinctly communicate what people can select?

**Comment by meiqimichelle** [comment]
Woot! As soon as we get words from Corey, we're good to go here, methinks!

File: index.html (hunk 1 above)
Comment (coreycaitlin) [inline index.html]: I did this:

screen shot 2016-09-15 at 10 25 30 am

**Comment by coreycaitlin** [comment]
I'm seeing weird padding on the explore/national page still @gemfarmer

**Comment by ericronne** [comment]
@coreycaitlin let's chat! ;)

**Comment by gemfarmer** [comment]
@coreycaitlin where is the padding weirdness?

**Comment by gemfarmer** [comment]
Also, I like the new wording!

**Comment by coreycaitlin** [comment]
@gemfarmer Here's the weirdness:

screen shot 2016-09-15 at 10 49 01 am

**Comment by gemfarmer** [comment]
@coreycaitlin oi!

**Comment by gemfarmer** [comment]
@coreycaitlin how about now?!

**Comment by ericronne** [comment]
@coreycaitlin thanks 👍 

Digging the top label, but "state or offshore region" feels a little long to me. Also it isn't technically accurate, right? (Since you could also choose national data.) I wanted to simply show "Nationwide data" as the default, but i think that's cumbersome to code (and a little odd from an interaction standpoint). Any more general option we could drop in there?

**Comment by coreycaitlin** [comment]
@ericronne Here are a few of the options I considered:

- `National, state, or offshore` — way too long and somehow more confusing?
- `Choose a state or offshore region` — way too long, but yay verb!
- `Location` or `Choose location` — technically accurate, but it feels pretty cryptic; "location" is another layer of abstraction up, and doesn't tell people what they can expect to see in the list
- `State or offshore region` — not technically _quite_ right, since "nationwide" is also in the list, but it felt least bad because it tells people what to expect; since "nationwide" is first, I'm not too worried about people missing it if they do click

My priorities here are:

- Getting verbs ("choose" and "explore") into either the label or the default
- Telling people what to expect when they click (i.e. a list of states and offshore regions, mostly)
- Brevity

I think either `Choose location` or `State or offshore region` could work; if we go with `Choose location` I'll to re-write the content immediately above to remove "location."

**Comment by ericronne** [comment]
You are a killer content designer. Just sayin.

Loving "Choose location"
Could it be as simple as "Explore data" / "Choose location"??

**Comment by ericronne** [comment]
@coreycaitlin also if you dig in there to make updates, could you enlarge the first label ("Explore data:" or wherever we land) to 1rem? thx!

**Pull request merged**